pypit/scienceimage.py

```diff
@@ -102,13 +96,15 @@ class ScienceImage(processimages.ProcessImages):
 
     """
     # Keep order same as processimages (or else!)
-    def __init__(self, file_list=[], spectrograph=None, settings=None,
-                 tslits_dict=None, tilts=None, det=None, setup=None, datasec_img=None,
-                 bpm=None, maskslits=None, pixlocn=None, objtype='science',
-                 fitstbl=None, scidx=0):
+    def __init__(self, spectrograph, file_list=[], frame_par=None, trace_objects_par=None,
```

Comment:
I'd coordinate modifying this object with @jhennawi

Reply:
This class is going to be completely rewritten so I would not spend time modifying it now in its current form.